frontend/src/index.css

```diff
@@ -0,0 +1,4 @@
+*{
+    margin:0;
+    font-family: 'DM Sans', sans-serif;
```

Comment:
Add DM Sans file and `@font-face` to `frontend/src/css/fonts.css` and then import `fonts.css` here.